src/lib/jsonParseV2/parseAttributeValue.cpp

```diff
@@ -49,7 +49,8 @@ std::string parseAttributeValue(ConnectionInfo* ciP, ContextAttribute* caP)
   if (document.HasParseError())
   {
     alarmMgr.badInput(clientIp, "JSON parse error");
-    oe.fill(SccBadRequest, "Errors found in incoming JSON buffer");
+    oe.reasonPhrase = ERROR_STRING_PARSERROR;
```

Comment:
Or simply use the old fill() but with this new define:

    oe.fill(SccBadRequest, ERROR_STRING_PARSERROR);

Isn't that what you want?

BTW, what about oe.code?  Not calling fill, just setting the reasonPhrase might not be enough ...